namespaces/live-1.cloud-platform.service.justice.gov.uk/track-a-move-dev/00-namespace.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: track-a-move-dev
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/slack-channel: "hmpps_dev"
+    cloud-platform.justice.gov.uk/application: "Track a Move"
+    cloud-platform.justice.gov.uk/owner: "HMPPS: pecs-digital-tech@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/hmpps-book-secure-move-api"
```

Comment:
This project does not have a repo (yet) so this is referring to it's parent project.